### PR TITLE
Feature/better errors

### PR DESCRIPTION
**Pull request by chrishunter-konsentus**
Previously the errors just stated that the branch was blocked. Since this didn't happen often, it generated questions directed to myself and others which required some research into what was blocked.

image

The proposed solution is for the error to be clearer as to what is actually being blocked, and who to contact around getting it resolved.

image

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -73,25 +73,29 @@ isBranchBlocked() {
         return 0
     fi
 
-    # target branch is considered blocked if its current HEAD is ahead of the next branch HEAD
-    # ie the head of the 'target' branch is not an ancester of the 'after target' branch
+    target branch is considered blocked if its current HEAD is ahead of the next branch HEAD
+    ie the head of the 'target' branch is not an ancester of the 'after target' branch
```
Comment (jonmoss-konsentus) [inline entrypoint.sh]: I think the comments have been uncommented by mistake.

**Comment by jonmoss-konsentus** [comment]
The message returned is much better than before

File: entrypoint.sh (hunk 1 above)
Comment (chrishunter-konsentus) [reply entrypoint.sh]: Ah, block unquote as I was forcing failures. Thanks!

File: entrypoint.sh [hunk 2]
```diff
@@ -77,21 +77,25 @@ isBranchBlocked() {
     # ie the head of the 'target' branch is not an ancester of the 'after target' branch
     if git merge-base --is-ancestor \
         $(git rev-parse origin/${TARGET_BRANCH}) \
-        $(git rev-parse origin/${AFTER_TARGET_BRANCH})
-    then
+        $(git rev-parse origin/${AFTER_TARGET_BRANCH}); then
```
Comment (davidjdixon) [inline entrypoint.sh]: as a general stylistic item, i would prefer to keep the "then" on an alternate line as it makes it easier to see the difference between the condition and the body

**Review by davidjdixon — CHANGES_REQUESTED** [review]
just one minor change, but rest looks great to me!

File: entrypoint.sh (hunk 2 above)
Comment (chrishunter-konsentus) [reply entrypoint.sh]: Appears to be an autoformat thing going on here, let me see if it will let me reverse it

Comment (chrishunter-konsentus) [reply entrypoint.sh]: @davidjdixon Not sure what formatting rules are doing this, but it should be reverted now.

**Review by davidjdixon — APPROVED** [review]
Woot woot!

**Pull request merged**